pynest/nest/__init__.py

```diff
@@ -198,6 +202,40 @@ def init(argv):
         _kernel.NESTError("Initiatization of NEST failed.")
 
 
+def expand_sli_path():
+    """ Add paths defined in SLI_PATH to searchpath. """
+
+    sli_paths = [path for path in
+                 os.environ.get("SLI_PATH", "").split(os.path.pathsep)
+                 if path != ""]
+
+    assert_sli_valid(sli_paths)
+
+    for path in sli_paths:
+        sli_run("({}) addpath".format(path))
+
+
+def assert_sli_valid(names):
+    for n in names:
+        if re.search(r"[\(\)\\]", n):
+            raise _kernel.NESTError("Path or Module names must not contain "
+                "parentheses or backslashes.")
+
+
+
+def load_modules():
+    """ Load a list of modules that are specified via NEST_MODULES. """
+
+    modules = [module for module in
+               os.environ.get("NEST_MODULES", "").split(os.path.pathsep)
+               if module != ""]
+
+    assert_sli_valid(modules)
+
+    for module in modules:
+        sli_run("({}) Install".format(module))
+
+
```

Comment:
Extra blank here ("pynest/nest/__init__.py:225:1: E303 too many blank lines (3)" -- I think pep8 is pointing at the wrong line